source/TagRestrictions/TagsChecker.cs

```diff
@@ -328,5 +379,15 @@ private static string NameForTag(string tag)
 
             return tag;
         }
+
+        private static string NameForTags(IEnumerable<string> tags)
+        {
+            string result = "";
+            foreach (var tag in tags)
+            {
+                result = (result.Length == 0) ? $"{NameForTag(tag)}" : $"{result}, {NameForTag(tag)}";
+            }
+            return result;
```

Comment:
maybe return `string.Join(", ", tags.Select(x => NameForTag(x)));` to keep it concise